Makefile

```diff
@@ -1,4 +1,4 @@
 all     :; dapp --use solc:0.6.12 build
 clean   :; dapp clean
-test    :; dapp --use solc:0.6.12 test --rpc -v
+test    :; dapp --use solc:0.6.12 test --rpc -v --fuzz-runs $(runs)
```

Comment:
This should have a default so we can run `make test` without having to specify runs.

Reply:
Done.